packages/vaadin-date-picker/src/vaadin-date-picker.js

```diff
@@ -3,17 +3,24 @@
  * Copyright (c) 2021 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { PolymerElement, html } from '@polymer/polymer';
 import { GestureEventListeners } from '@polymer/polymer/lib/mixins/gesture-event-listeners.js';
-import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 import '@polymer/iron-media-query/iron-media-query.js';
-import { ControlStateMixin } from '@vaadin/vaadin-control-state-mixin/vaadin-control-state-mixin.js';
 import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { AriaLabelMixin } from '@vaadin/field-base/src/aria-label-mixin.js';
+import { ClearButtonMixin } from '@vaadin/field-base/src/clear-button-mixin.js';
+import { FieldAriaMixin } from '@vaadin/field-base/src/field-aria-mixin.js';
+import { InputConstraintsMixin } from '@vaadin/field-base/src/input-constraints-mixin.js';
+import { InputSlotMixin } from '@vaadin/field-base/src/input-slot-mixin.js';
+import '@vaadin/vaadin-date-picker/src/vaadin-date-picker-overlay.js';
+import '@vaadin/vaadin-date-picker/src/vaadin-date-picker-overlay-content.js';
+import '@vaadin/input-container/src/vaadin-input-container.js';
+import { inputFieldShared } from '@vaadin/text-field/src/vaadin-input-field-shared-styles.js';
```

Comment:
Let's extract this change to the separate PR and update all field components.

I think we can do it this way:

1. Move `@vaadin/text-field/src/vaadin-input-field-shared-styles.js` to `@vaadin/field-base`
2. Move `lumo-input-field-shared-styles` to Lumo, and same for Material
3. Remove `@vaadin/text-field` dependency from combo-box, select and date-picker.

Reply:
Done